src/_locales/en/messages.json

```
@@ -108,10 +108,10 @@
     "message": "Just a few quick steps to connect the extension to your wallet"
   },
   "connectWallet_label_walletAddress": {
-    "message": "Enter wallet address/payment pointer"
+    "message": "Enter your wallet address/payment pointer"
   },
   "connectWallet_labelGroup_amount": {
-    "message": "Enter amount to allocate from wallet"
+    "message": "Enter the amount to allocate from your wallet."
```

For consistency: the label for `walletAddress` doesn't have a period.

```suggestion
    "message": "Enter the amount to allocate from your wallet"
```